profiles/Dun-sin.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/78784850/168474890-5ab91178-c9aa-4fdd-8294-484613272de5.png",
+  "issueId": 1124,
+  "name": "Dunsin",
+  "username": "Dun-sin"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```